docs/src/token-2022.md

```diff
@@ -8,9 +8,12 @@ fungible and non-fungible tokens.
 The Token-2022 Program is a superset of the functionality provided by the
 [Token Program](token.mdx).
 
-The program is still under audit and not meant for full production use. In the
-meantime, all clusters have the latest program deployed **for testing and development
-purposes ONLY**.
+The program is ready for full production use. All clusters have the latest
+program deployed **without confidential transfer functionality**.
+
+The program with confidential transfer functionality will be deployed once
+Solana v1.17 reaches mainnet-beta with the appropriate syscalls enabled. See the
```

Comment:
replicode? maybe just replace with a spruced up link out to the status?

Reply:
That's a good point... We had it here as extra cover-your-ass to be really clear that it wasn't ready for prod. we could just remove the whole thing, what do you think?